src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -51,28 +52,33 @@ export default function DownloadDetails(props: DownloadDetailsProps) {
   }, [accessToken])
 
   const timeEstimateInSeconds =
-    isLoading || downloadSpeed === 0 ? 0 : numBytes / downloadSpeed
+    isLoading || downloadSpeed === 0 || !numBytes ? 0 : numBytes / downloadSpeed
+  const isTimeEstimateLoading = timeEstimateInSeconds === 0
   const friendlyTime =
-    timeEstimateInSeconds === 0
+    isTimeEstimateLoading
       ? ''
       : moment.duration(timeEstimateInSeconds, 'seconds').humanize()
   const numBytesTooltipId = 'num_bytes_id'
   const friendlyTimeTooltipId = 'friendly_time_id'
-  const isInactive = numFiles === 0 || timeEstimateInSeconds === 0
-  const iconClassName = isInactive ? 'SRC-inactive' : 'SRC-primary-text-color'
+
+  const isZeroFiles = numFiles === 0
+  const fileCountIconClass = isZeroFiles ? 'SRC-inactive' : 'SRC-primary-text-color'
+
+  const timeEstimateIconClass = isTimeEstimateLoading ? 'SRC-inactive' : 'SRC-primary-text-color'
   return (
     <span className="download-details-container">
       <span>
-        <FontAwesomeIcon className={iconClassName} icon="file" />
-        {isInactive ? (
+        <FontAwesomeIcon className={fileCountIconClass} icon="file" />
+        {isZeroFiles ? (
           <SkeletonInlineBlock width={50} />
         ) : (
           <> {numFiles}&nbsp;files </>
         )}
       </span>
-      <span
+      {numBytes && <span
```

Comment:
only show the number of bytes if set (and greater than 0, since 0 is also falsy)